### PR TITLE
test: Add initial e2e tests for default owner setup

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: cypress/pages/modals/message-box.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { BasePage } from "../base";
+
+export class MessageBox extends BasePage {
+	getters = {
+		modal: () => cy.get('.el-message-box', { withinSubject: null }),
```
Comment (mutdmour) [inline cypress/pages/modals/message-box.ts:5]: could not find a non-hacky way to add test ids to element ui components

Comment (alexgrozav) [reply cypress/pages/modals/message-box.ts:5]: I suppose we'll have to compromise on this one for `element-ui`.

**Review by alexgrozav — APPROVED** [review]
Awesome work, @mutdmour!

**Comment by janober** [comment]
Got released with `n8n@0.204.0`

**Pull request merged**